contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -476,6 +471,66 @@ contract GetValidatorConfig is DualAggregatorBaseTest {
   }
 }
 
+contract ValidateAnswer is ConfiguredDualAggregatorBaseTest {
+  function test_RevertIf_InsufficientGas() public {
+    ReportGenerator.SignedReport memory signedReport1 =
+      s_reportGenerator.generateSignedReport(1, uint32(block.timestamp));
+    AggregatorValidator newValidator = new AggregatorValidator();
+
+    uint256 gas = gasleft();
+    s_aggregator.setValidatorConfig(newValidator, uint32(gas));
+
+    _changePrank(s_aggregator.getTransmitters()[0]);
+    vm.expectRevert(DualAggregator.InsufficientGas.selector);
+    s_aggregator.transmit(
+      signedReport1.reportContext, signedReport1.report, signedReport1.rs, signedReport1.ss, signedReport1.rawVs
+    );
+
+    (uint256 previousRoundId, int256 previousAnswer, uint256 currentRoundId, int256 currentAnswer) =
+      newValidator.getLatestValidatedValues();
+
+    vm.assertEq(previousRoundId, 0, "previous round id not zero");
+    vm.assertEq(previousAnswer, 0, "previous answer not zero");
+    vm.assertEq(currentRoundId, 0, "current round id not zero");
+    vm.assertEq(currentAnswer, 0, "current answer not zero");
+  }
+
+  function test_NotToRevert_ValidatorZeroAddress() public {
```

Comment:
remove _NotToRevert_ . If this test case should pass you can name it something like test_ValidateWithNonContractAddress